documentation/bibliography.bib

```diff
@@ -780,6 +780,48 @@ @TechReport{Westra2017
 month = sep
 }
 
+@Article{Bulirsch1965,
+author="Bulirsch, Roland",
```

Comment:
In the generated PDF, we do; in the .bib file, it is best to have as much information as possible, in case we want to switch to an unabbreviated style.